Update readme references and examples (#123)
ethereumjs/merkle-patricia-tree: README.md

test()
```

> Additional examples with detailed explanations are available in [this repository](https://github.com/gabrocheleau/merkle-patricia-trees-examples).

Reviewer: ```suggestion

Additional examples with detailed explanations are available in [this repository](https://github.com/gabrocheleau/merkle-patricia-trees-examples).
```

test()
```

> Additional examples with detailed explanations are available in [this repository](https://github.com/gabrocheleau/merkle-patricia-trees-examples).

Reviewer: ```suggestion
Additional examples with detailed explanations are available [here](https://github.com/gabrocheleau/merkle-patricia-trees-examples).
```

Just a nitpick.
